### PR TITLE
Upload images to IPFS

### PR DESCRIPTION
**Pull request by ktalanda**

### CODE REVIEW

File: client/src/ipfsUploader.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { create } from 'ipfs-http-client'
+
+export const uploadToIpfs = async (file) => {
+    const client = create('https://localhost:5001')
```
Comment (bjing) [inline client/src/ipfsUploader.js]: where's port `5001` configured?

Comment (bjing) [reply client/src/ipfsUploader.js]: What I'm concerned is: will this port change due to version update?

When we initiate ipfs node, is it always gonna be port 5001, can we specify the port 5001 to make sure this is the case?

**Review by bjing — APPROVED** [review]

File: client/src/ipfsUploader.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    try {
+        console.log("IPFS: START uploading")
```
Comment (bjing) [inline client/src/ipfsUploader.js]: Better print out the file name that you're uploading, unless the client.add call prints it.

File: client/src/ipfsUploader.js (hunk 1 above)
Comment (ktalanda) [reply client/src/ipfsUploader.js]: I'm still exploring IPFS :)
I will definitely check how to set the port number to make it more robust, seems fragile, good catch.

**Pull request merged**